generators/app/utils.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const chalk = require('chalk');
+
+module.exports = {
+    noConfig: function (name, configs) {
+        return function (values) {
+            return new Promise(function (resolve, reject) {
+                var value = configs[name];
+                resolve(value == null || value == undefined);
+            })
+        }
+
+    },
+    configInfo: function (config) {
```

Comment:
Can you inline this into the generator and use `this.log`. Also please use single quotes instead of doubles, you can verify this by running `npm run lint`